src/generators/python/index.ts

```diff
@@ -18,3 +18,7 @@ export {
   DefaultPropertyKeyConstraints as PythonDefaultPropertyKeyConstraints,
   defaultPropertyKeyConstraints as pythonDefaultPropertyKeyConstraints
 } from './constrainer/PropertyKeyConstrainer';
+
+export {
+  PYTHON_PYDANTIC_PRESET
+} from  './presets/Pydantic';
```

Comment:
Add this to the `src/generators/python/presets/index.ts` file instead 🙂 Then it's automatically exported.

Reply:
done